vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -96,9 +96,16 @@ vic_cesm_init(vic_clock     *vclock,
     // populate model state, either using a cold start or from a restart file
     vic_populate_model_state(trim(cmeta->starttype));
 
+    // initialize forcings
+    vic_force();
+
     // initialize output structures
     vic_init_output(&dmy_current);
 
+    // assign case name to state file name
+    strncpy(filenames.statefile, trim(cmeta->caseid),
+            sizeof(filenames.statefile));
+
```

Comment:
please put this in `vic_cesm_start()`.  As you can see, we don't really do anything in this top level function except call other functions and I'd like to keep it that way.